components/menu/index.css

```diff
@@ -369,11 +369,11 @@ governing permissions and limitations under the License.
   display: grid;
   /* stylelint-disable declaration-block-no-redundant-longhand-properties */
   grid-template-areas:
-    ".            chevronAreaCollapsible .             iconArea sectionHeadingArea  .         .           .                 "
+    ".            chevronAreaCollapsible .             headingIconArea  sectionHeadingArea  .         .           .                 "
```

Comment:
this previously had two `iconAreas` which causes some issues with menu item icons and labels not being on the correct row with in the grid.